crates/chain/src/tx_graph.rs

```diff
@@ -127,7 +146,7 @@ impl Default for TxNodeInternal {
     }
 }
 
-/// An outwards-facing view of a transaction that is part of the *best chain*'s history.
+/// A transaction that is included in a certain chain.
```

Comment:
Since a canonical transaction could still be in the mempool, it might be good to mention it here as well.